CondFormats/L1TObjects/interface/L1TMuonBarrelKalmanParams.h

```diff
@@ -0,0 +1,96 @@
+///
+/// \class L1TMuonBarrelKalmanParams
+///
+/// Description: Placeholder for Kalman BMTF parameters
+///
+///
+/// \author: Panos Katsoulis
+///
+
+#ifndef L1TBMTFKalmanParams_h
+#define L1TBMTFKalmanParams_h
+
+#include <memory>
+#include <iostream>
+#include <vector>
+
+#include "CondFormats/Serialization/interface/Serializable.h"
+#include "CondFormats/L1TObjects/interface/LUT.h"
+
+class L1TMuonBarrelKalmanParams {
+public:
+  L1TMuonBarrelKalmanParams();
+  ~L1TMuonBarrelKalmanParams() {}
+
+  enum { Version = 1 };
+  enum { CONFIG = 0, NUM_BMTF_PARAM_NODES = 2 };
+
+  class Node {
+  public:
+    std::string type_;
+    std::string kalmanLUTsPath_;
+    unsigned fwVersion_;
+    l1t::LUT LUT_;
+    COND_SERIALIZABLE;
+  };
+
+  // for future extention
```

Comment:
Indeed we made clear during the meeting that once the CondFormat/Record is in the release it can not be changed.
That's why I put the hold command, to make sure everything is as it is supposed to be.
We can also request to remove such comments as "For future extention".

Reply:
No, we told Panos about the policy of no change in the cond format today, he said he'll come back to us regarding this.

Reply:
thx for holding it
yes indeed I will update it today, sorry yesterday I couldn't make it

Reply:
Thanks Panos! Also please, as Marco suggested, remove all the comments saying "for future extention" and similar!

Reply:
> thx for holding it
> yes indeed I will update it today, sorry yesterday I couldn't make it

please bear in mind that the deadline for the last open pre-release of 12_0_X is today. If you want to have this in 12_0 you should have it ready by today, and hopefully (re)-approved by tomorrow morning at most

Reply:
thx, I'll put full focus on this